CIP-1694/README.md

```diff
@@ -491,7 +502,7 @@ for example, a motion of no confidence is enacted.
 | Action                                           | Description |
 | :---                                             | :--- |
 | 1. Motion of no-confidence                       | A motion to create a _state of no-confidence_ in the current constitutional committee |
-| 2. New constitutional committee and/or threshold | Changes to the members of the constitutional committee and/or to its signature threshold |
+| 2. New constitutional committee and/or threshold | Changes to the members of the constitutional committee and/or to its signature threshold and/or term limits|
```

Comment:
As term limits are now explicitly mentioned, shouldn't the governance action name also include term limits? 

>  **New constitutional committee and/or threshold and/or term limits**